packages/synapse-react-client/src/components/AccessRequirement/ImposeRestrictionDialog/ImposeRestrictionDialog.tsx

```diff
@@ -44,24 +44,26 @@ export default function ImposeRestrictionDialog(
     boolean | undefined
   >(undefined)
 
-  const { mutate: createLockedAccessRequirement, isLoading } =
-    useCreateLockAccessRequirement({
-      onSuccess: () => {
-        displayToast('Successfully imposed restriction', 'success')
-        // PORTALS-2664: Send the user to the the ACT Service Desk
-        // so they can tell ACT more information about what kind of
-        // Conditions For Use (or Data Access Restriction) should be
-        // added.
-        window.open(
-          'https://sagebionetworks.jira.com/servicedesk/customer/portal/8/group/15/create/134',
-          '_blank',
-        )
-        onClose()
-      },
-      onError: e => {
-        displayToast(`Failed to impose restriction: ${e.reason}`, 'danger')
-      },
-    })
+  const {
+    mutate: createLockedAccessRequirement,
+    isPending: createLockedARIsPending,
```

Comment:
`useMutation`'s `isLoading` was renamed to `isPending`, so make that change, and in most cases rename the variables to match. I think it's a much better name for the mutation state.